pkgs/faf-client/default.nix

```diff
@@ -0,0 +1,97 @@
+{
+  lib,
+  stdenvNoCC,
+  makeWrapper,
+  makeDesktopItem,
+  # regular jdk doesnt work due to problems with JavaFX even with .override { enableJavaFX = true; }
+  openjdk17-bootstrap,
+  xorg,
+  libGL,
+  gtk3,
+  glib,
+  alsa-lib,
+  unstable ? false,
+}: let
+  pname = "faf-client";
+
+  versionStable = "2023.1.2";
+  sha256Stable = "0lfyfbvjzfgkq3b485d8ma8fjc3hbc5p5h2ffl3lrvai6ksa59g2";
+  srcStable = builtins.fetchTarball {
+    url = "https://github.com/FAForever/downlords-faf-client/releases/download/v${versionStable}/faf_unix_${builtins.replaceStrings ["."] ["_"] versionStable}.tar.gz";
+    sha256 = sha256Stable;
+  };
+
+  versionUnstable = "2023.2.0-alpha-1";
+  sha256Unstable = "0q09p1qgxq80zm64miajxk2q8fp56s5641kbvygny1arjdfbrs9r";
+  srcUnstable = builtins.fetchTarball {
+    url = "https://github.com/FAForever/downlords-faf-client/releases/download/v${versionUnstable}/faf_unix_${builtins.replaceStrings ["."] ["_"] versionUnstable}.tar.gz";
+    sha256 = sha256Unstable;
+  };
+
+  meta = with lib; {
```

Comment:
You should add yourself as a maintainer here.

Reply:
I will add myself when I get a first commit to nixpkgs